libs/api/domains/cms/src/lib/cms.elasticsearch.service.ts

```diff
@@ -175,6 +177,27 @@ export class CmsElasticsearchService {
     return response ? JSON.parse(response) : null
   }
 
+  async getOrganizationSubpage(
```

Comment:
Should this maybe be called `getSingleOrganizationSubpage` to follow convention?